File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.cs
using System;

public class Solution {
    public int[] solution(string myString) {
        string[] s = myString.Split("x");
        int[] answer = new int[s.Length];
        for(int i = 0; i < answer.Length ; i++)
        {
            answer[i] = s[i].Length;
        }
        return answer;
    }
}

File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.cpp
#include <string>
#include <vector>
#include <sstream>
using namespace std;

vector<int> solution(string myString) {
    vector<int> answer;
    string temp = myString + "x";
    stringstream ss(temp);
    
    while(getline(ss,temp,'x'))
        answer.push_back(temp.size());
    
    return answer;
}